tests/Unit/Domain/DomainCreators/Test_Shell.cpp

```diff
@@ -418,3 +418,29 @@ SPECTRE_TEST_CASE("Unit.Domain.DomainCreators.Shell.Factory.ShellWedges",
         which_wedges);
   }
 }
+
```

Comment:
Please make an issue to reduce the number of SPECTRE_TEST_CASEs in this file by having multiple test functions that are called in one SPECTRE_TEST_CASE

Also add a test that checks that you get the same radii for Element boundaries using one layer at refinement level 3 and 8 layers using refinement level 0 for both linear and log radial maps